stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/textinput/StreamMessageInputView.kt

```diff
@@ -0,0 +1,165 @@
+                        R.styleable.StreamMessageInputView_attachmentButtonIconPressedColor,
+                        ContextCompat.getColor(context, R.color.stream_gray_light)
+                    )
+                )
+            )
+
+            layoutParams.width = typedArray.getDimensionPixelSize(
+                R.styleable.StreamMessageInputView_attachmentButtonWidth,
+                context.resources.getDimensionPixelSize(R.dimen.stream_attachment_button_width)
+            )
+
+            layoutParams.height = typedArray.getDimensionPixelSize(
+                R.styleable.StreamMessageInputView_attachmentButtonHeight,
+                context.resources.getDimensionPixelSize(R.dimen.stream_attachment_button_height)
+            )
+        }
+    }
+
+    private fun configLightningButton(typedArray: TypedArray) {
+        binding.ivOpenEmojis.run {
+            isVisible =
+                typedArray.getBoolean(R.styleable.StreamMessageInputView_showLightningButton, true)
+
+            typedArray.getDrawable(R.styleable.StreamMessageInputView_lightningButtonIcon)
+                ?.let(this::setImageDrawable)
+
+            DrawableCompat.setTintList(
+                drawable,
+                getColorList(
+                    typedArray.getColor(
+                        R.styleable.StreamMessageInputView_lightningButtonIconColor,
+                        ContextCompat.getColor(context, R.color.stream_gray_dark)
+                    ),
+                    typedArray.getColor(
+                        R.styleable.StreamMessageInputView_lightningButtonIconPressedColor,
+                        ContextCompat.getColor(context, R.color.stream_white)
+                    ),
+                    typedArray.getColor(
+                        R.styleable.StreamMessageInputView_lightningButtonIconPressedColor,
+                        ContextCompat.getColor(context, R.color.stream_gray_light)
+                    )
+                )
+            )
+
+            layoutParams.width = typedArray.getDimensionPixelSize(
+                R.styleable.StreamMessageInputView_lightningButtonWidth,
+                context.resources.getDimensionPixelSize(R.dimen.stream_attachment_button_width)
+            )
+
+            layoutParams.height = typedArray.getDimensionPixelSize(
+                R.styleable.StreamMessageInputView_lightningButtonHeight,
+                context.resources.getDimensionPixelSize(R.dimen.stream_attachment_button_height)
+            )
+        }
+    }
+
+    private fun showSendMessageEnabled() {
+        binding.ivSendMessage.setImageResource(R.drawable.stream_ic_filled_up_arrow)
+    }
+
+    private fun hideSendMessageEnabled() {
+        binding.ivSendMessage.setImageResource(R.drawable.stream_ic_filled_right_arrow)
+    }
+
+    private fun configTextInput(typedArray: TypedArray) {
+        TextInputHandler(binding.etMessageTextInput, ::showSendMessageEnabled, ::hideSendMessageEnabled)
```

Comment:
Seems like this class is not doing much. What about replacing it with 
`binding.etMessageTextInput.addTextChangedListener(onTextChanged = {  })`